davinci_api-pingone_protect-reg-authn-pkg/terraform/vars.tf

```diff
@@ -22,6 +22,11 @@ variable "admin_env_id" {
   description = "P1 Environment containing the Worker App"
 }
 
+variable "PINGONE_ENVIRONMENT_ID" {
+  type        = string
+  description = "P1 Administrator to assign Roles to"
+}
+
```

Comment:
Is the code at bottom of file still needed?  It looks like Patrick replaced it with the utils module, i.e.:
old: value = "https://auth.pingone.${local.pingone_domain}"
new: value = module.pingone_utils.pingone_url_auth_path